examples/backtick-templates/backtick_templates.out

```diff
@@ -1,3 +1,4 @@
 $ bal run backtick_templates.bal
 Hello, James
 Backtick:`
+A string-template-expr is evaluated by evaluating the expression in each interpolation in the order in which they occur.
```

Comment:
Seems the string is not split into a new line

Reply:
ignore the comment please.

Reply:
On a second thought, I feel having newline also in a middle of the string template and showing the output is also worth, as it is useful for examples like email content..

Reply:
The string `s2` here, https://ballerina.io/learn/by-example/backtick-templates/. Isn't it kind of similar to using special characters like new line in a string template.